src/helper/tags.hpp

```diff
@@ -0,0 +1,19 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#pragma once
+
+namespace dds::tag {
+
+constexpr const char *event_rules_loaded = "_dd.appsec.event_rules.loaded";
```

Comment:
This could be optimized so you're not allocating new strings for the tag and metric keys on every request. This is trivial if you're only using const strings as keys, but gets a little more tricky if you want to mix owning and non-owning strings.